saleor/static/dashboard-next/translations/components/TranslationFields/TranslationFields.tsx

```diff
@@ -163,6 +164,7 @@ const TranslationFields = withStyles(styles, { name: "TranslationFields" })(
                           edit={false}
                           initial={field.value}
                           saveButtonState="default"
+                          onDiscard={() => navigate("?", true)}
```

Comment:
It's visual component and callback `onDiscard` should be passed down from `TranslationsXXX` view through `TranslationsXXXPage` component